ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs

```diff
@@ -8,11 +8,6 @@ static class ViewManagersPropertyCache
     {
         private static readonly IReadOnlyDictionary<string, IPropertySetter> s_shadowEmpty = new Dictionary<string, IPropertySetter>();
 
-        private static readonly object[] s_viewArgsBuffer = new object[2];
```

Comment:
We can move this back under UIManager in my pull request if you like, as I've already move it to the Shared project under /Views.